output/schema/schema.json

```diff
@@ -18330,9 +18361,14 @@
           "stability": "stable"
         }
       },
-      "description": "Deletes a repository.",
+      "description": "Delete snapshot repositories.\nWhen a repository is unregistered, Elasticsearch only removes the reference to the location where the repository is storing the snapshots.\nThe snapshots themselves are left untouched and in place.",
```

Comment:
```suggestion
      "description": "Delete snapshot repositories.\nWhen a repository is unregistered, Elasticsearch removes only the reference to the location where the repository is storing the snapshots.\nThe snapshots themselves are left untouched and in place.",
```

Reply:
Fixed in https://github.com/elastic/elasticsearch-specification/pull/3314/commits/dc70256ca197d43bd3796691a02dc19e2a9a13f7